Untitled-1.js

```diff
@@ -0,0 +1,25 @@
+
+
+
+class Person {
+   constructor(attributes) {
+     this.age = attributes.age;
+     this.name = attributes.name;
+   }
+   standUp(channel) {
+       return `${this.name} announces to ${channel}, @channel standy times!`;
+     }
+ }
+
+ class ProjectManager extends Person {
+   constructor(pmAttributes) {
```

Comment:
I think you started this and didn't finish. after your super you would want to do like "this.classname = props.classname;" and so on